docs/user/02-configuration.md

```diff
@@ -40,40 +40,64 @@ For high availability, the NATS servers must be set up across different availabi
-| **resources.&#x200b;limits**                               | map\[string\]\{integer or string\} | Limits describes the maximum amount of compute resources allowed. Fore more information, check [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).                                                                                                                                                                                |
-| **resources.&#x200b;requests**                             | map\[string\]\{integer or string\} | Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. Requests cannot exceed Limits. Fore more information, check [Resource Management for Pods and Containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/). |
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **annotations**  | map\[string\]string | Annotations allows to add annotations to NATS. |
+| **cluster**  | object | Cluster defines configurations that are specific to NATS clusters. |
+| **cluster.&#x200b;size**  | integer | Size of a NATS cluster, i.e. number of NATS nodes. |
+| **jetStream**  | object | JetStream defines configurations that are specific to NATS JetStream. |
+| **jetStream.&#x200b;fileStorage**  | object | FileStorage defines configurations to file storage in NATS JetStream. |
+| **jetStream.&#x200b;fileStorage.&#x200b;size**  | \{integer or string\} | Size defines the file storage size. |
+| **jetStream.&#x200b;fileStorage.&#x200b;storageClassName**  | string | StorageClassName defines the file storage class name. |
+| **jetStream.&#x200b;memStorage**  | object | MemStorage defines configurations to memory storage in NATS JetStream. |
+| **jetStream.&#x200b;memStorage.&#x200b;enabled**  | boolean | Enabled allows the enablement of memory storage. |
+| **jetStream.&#x200b;memStorage.&#x200b;size**  | \{integer or string\} | Size defines the mem. |
+| **labels**  | map\[string\]string | Labels allows to add Labels to NATS. |
+| **logging**  | object | JetStream defines configurations that are specific to NATS logging in NATS. |
+| **logging.&#x200b;debug**  | boolean | Debug allows debug logging. |
+| **logging.&#x200b;trace**  | boolean | Trace allows trace logging. |
+| **resources**  | object | Resources defines resources for NATS. |
+| **resources.&#x200b;claims**  | \[\]object | Claims lists the names of resources, defined in spec.resourceClaims,
+that are used by this container.
+
+This is an alpha field and requires enabling the
+DynamicResourceAllocation feature gate.
+
```

Comment:
something went wrong with the formatting at this point, could you have another look, please?